ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -102,11 +103,10 @@ public void SetFontStyle(string fontStyleValue)
         [ReactProp(ViewProps.LetterSpacing)]
         public void SetLetterSpacing(int letterSpacing)
         {
-            var spacing = 50*letterSpacing; // TODO: Find exact multiplier (50) to match iOS
-
-            if (_letterSpacing != spacing)
+            var fontStretch = FontStyleHelpers.ConvertFontStretch(letterSpacing);
```

Comment:
I have made some assumptions here about the letterSpacing that need to be investigated further...